librz/core/cmd_descs/cmd_meta.yaml

```diff
@@ -0,0 +1,341 @@
+# SPDX-FileCopyrightText: 2021 RizinOrg <info@rizin.re>
+# SPDX-License-Identifier: LGPL-3.0-only
+---
+name: cmd_meta
+commands:
+  - name: C
+    summary: Show/Remove any kind of metainformation
+    subcommands:
```

Comment:
just remove this. Otherwise you have `C` inside `C` inside `C`.

Reply:
Do you mean that I should make `C`, `C.`, `C-`, etc top-level commands?

Reply:
yes